### PR TITLE
Add more types to conversion table

### PR DESCRIPTION
**Pull request by hramezani**
Closes https://github.com/pydantic/pydantic/issues/5363

Selected Reviewer: @Kludex

### CODE REVIEW

**Comment by hramezani** [comment]
please review

**Comment by Kludex** [comment]
I'll take some hours to review this, fyi

**Comment by hramezani** [comment]
> I'll take some hours to review this, fyi

No worries. It takes me hours to add them :)

File: docs/plugins/conversion_table.py [hunk 1]
```diff
@@ -820,4 +807,700 @@ class Row:
...
+        'Python',
+        valid_examples=[b'\x00\x00\x00\x00'],
+        core_schemas=[core_schema.PlainValidatorFunctionSchema],
+    ),
+    Row(
+        IPv4Address,
+        int,
+        'Lax',
+        'Python',
+        condition='integer representing the IP network, should be less than `2**32`',
```
Comment (idryzhov) [inline docs/plugins/conversion_table.py:1077]: `network` -> `address`?

File: docs/plugins/conversion_table.py [hunk 2]
```diff
@@ -820,4 +807,700 @@ class Row:
...
+        'Python',
+        valid_examples=[('192.168.0.1', '24')],
+        core_schemas=[core_schema.PlainValidatorFunctionSchema],
+    ),
+    Row(
+        IPv4Interface,
+        int,
+        'Lax',
+        'Python',
+        condition='integer representing the IP network, should be less than `2**32`',
```
Comment (idryzhov) [inline docs/plugins/conversion_table.py:1130]: `network` -> `address`?

File: docs/plugins/conversion_table.py [hunk 3]
```diff
@@ -820,4 +807,700 @@ class Row:
...
+        'Python',
+        valid_examples=[('2001:db00::1', '120')],
+        core_schemas=[core_schema.PlainValidatorFunctionSchema],
+    ),
+    Row(
+        IPv6Interface,
+        int,
+        'Lax',
+        'Python',
+        condition='integer representing the IP network, should be less than `2**32`',
```
Comment (idryzhov) [inline docs/plugins/conversion_table.py:1280]: `network` -> `address`?

`2**32` -> `2**128` for sure.

**Review by idryzhov — COMMENTED** [review]
A couple of comments on IP types.

**Comment by hramezani** [comment]
> A couple of comments on IP types.

Thanks @idryzhov. just fixed them

**Review by Kludex — APPROVED** [review]
I didn't review carefully.

**Pull request merged**